hokusai-app-updater/examples/detect-secrets-resource-class.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+_MATCH='      - image: artsy/detect-secrets:ci # pragma: allowlist secret'
+
+REPLACE='      - image: artsy/detect-secrets:ci # pragma: allowlist secret\n    resource_class: small'
+
+echo "replacing '$_MATCH' with '$REPLACE' in .circleci/config.yml"
+
+sed -i '' -e "s|$_MATCH|$REPLACE|" .circleci/config.yml
```

Comment:
Syntax is a little different than other examples here, but these changes were necessary to work with OSX's `sed`.